src/database/migrations/2020_09_19_170010_create_corona_status_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateCoronaStatusTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
```suggestion
    public function up(): void
```
Nurodyk, kad grąžinamas `void`. Šis grąžinimo tipas reiškia, kad realiai nebus grąžinta nieko, tik atlikta kažkokia logika metodo viduje (šiuo atveju, DB lentelės sukūrimas).

Apskritai, PHP 7.4 (dabartinėje) versijoje atsirado galimybės visur aprašyti kokių kintamųjų tipų tikimasi ir kokį tipą turi grąžinti metodas (funkcija), tad reikia stengtis visur nurodyti tipus bei kuo mažiau naudoti `php doc` - taip kodas bus lengviau skaitomas ir užims mažiau eilučių.